src/data_provider/tests/sysInfoPackagesLinuxHelper/sysInfoPackagesLinuxHelper_test.cpp

```diff
@@ -433,3 +434,108 @@ TEST_F(SysInfoPackagesLinuxHelperTest, parsePacmanInformationNull)
     EXPECT_EQ("Arch Linux", jsPackageInfo["vendor"]);
     EXPECT_EQ("", jsPackageInfo["description"]);
 }
+
+TEST_F(SysInfoPackagesLinuxHelperTest, parseApkNameKeyNotFound)
+{
+    std::vector<std::pair<char, std::string>> input;
```

Comment:
That is just the declaration, I think it would be useful if the variable was initialized with the same value in all cases. Also, I would need to revise all the tests for similar cases, and I found a case like this.
In one test I have
```c
    __alpm_list_t   mock            {};
    __alpm_pkg_t    data            {};
    __alpm_handle_t dataHandle      {};
    __alpm_list_t   dataFirstGroup  {};
    __alpm_list_t   dataSecondGroup {};
    __alpm_list_t   dataThirdGroup  {};
    __alpm_list_t   dataFourthGroup {};
 ``` 
 
 In another test I have
 ```c
    __alpm_list_t   mock        {};
    __alpm_pkg_t    data        {};
    __alpm_handle_t dataHandle  {};
    __alpm_list_t   dataGroups  {};
 ```
 I would have to include unused variables in the setup() or let some non-common declarations outside.